src/common/utils.lua

```diff
@@ -247,6 +247,41 @@ function utils.errorHandler(err)
 	return debug.traceback(err)
 end
 
+---@param outbox table
+---@description Validates the outbox
+---@example
+---```lua
+---utils.validateOutbox(ao.outbox)
+---```
+function utils.validateOutbox(outbox)
+	local messages = outbox.Messages
+	local spawns = outbox.Spawns
+	local assignments = outbox.Assignments
+
+	assert(type(messages) == "table" and type(spawns) == "table" and type(assignments) == "table")
+
+	for _, message in ipairs(messages) do
+		for _, tag in ipairs(message.Tags) do
+			local name, value = tag.name, tag.value
+			assert(type(name) == "string" and type(value) == "string", "Tag name and value must be strings")
+		end
+	end
+
+	for _, spawn in ipairs(spawns) do
+		for _, tag in ipairs(spawn.Tags) do
+			local name, value = tag.name, tag.value
+			assert(type(name) == "string" and type(value) == "string", "Tag name and value must be strings")
+		end
+	end
+
+	for _, assignment in ipairs(assignments) do
+		for _, tag in ipairs(assignment.Tags) do
+			local name, value = tag.name, tag.value
+			assert(type(name) == "string" and type(value) == "string", "Tag name and value must be strings")
+		end
+	end
+end
```

Comment:
i'd prefer asssertions in utils.Send that validate a message tags are all strings than wrapping this here

Reply:
yeah i think thats reasonable

Reply:
https://github.com/ar-io/ar-io-ant-process/commit/7366b5fb09f5c1ed39f241c701e1d2d7f7ca5726